content/tbb/export-and-import-bookmarks/contents.lr

```diff
@@ -0,0 +1,57 @@
+_model: question
+---
+title: How can I Export and Import Bookmarks?
+---
+seo_slug: export-and-import-bookmarks
+---
+description:
+
+Bookmarks in the Tor Browser can be exported, imported, backup, restored as well as imported from another browser.
+In order to export , import, backup or restore bookmarks in Tor browser, go to:
+
+- Hamburger-menu >> Library >> Bookmarks >> Show All Bookmarks (below the menu)
+- From the toolbar on the Library window, click Import and Backup
+
+**If you wish to export bookmarks**
+- Choose Export Bookmarks to HTML
+- In the Export Bookmarks File window that opens, choose a location to save the file, which is named bookmarks.html by default. The desktop is usually a good spot, but any place that is easy to remember will work.
+- Click the Savebutton. The Export Bookmarks File window will close.
+- Close the Library window.
+
+> Your bookmarks are now successfully exported from Tor. The bookmarks HTML file you saved is now ready to be imported into another web browser.
+
+**If you wish to Import bookmarks**
+- Choose Import Bookmarks from HTML
+- Within the Import Bookmarks File window that opens, navigate to the bookmarks HTML file you are importing and select the file.
+- Click the Open button. The Import Bookmarks File window will close.
+- Close the Library window.
+
+> The bookmarks in the selected HTML file will be added to your Tor bookmarks within the Bookmarks Menu directory.
+
+**If you wish to Backup**
+- Choose Backup
+- A new window opens and you have to choose the location to save the file. The file has a .json extension.
+
+**If you wish to Restore**
+- Choose Restore and then select the bookmark file you wish to restore. 
+- Click okay to the pop up box that appears and hurray, you just restored your backup bookmark.
+
+**Import Data from another browser**
+> Bookmarks can be transferred from Firefox to Tor. There are two ways to export and import bookmarks in Firefox: [HTML file](https://support.mozilla.org/en-US/kb/export-firefox-bookmarks-to-backup-or-transfer) or JSON file. Remember any custom bridges and search engine additions, too.
+After exporting the data from the browser, follow the above step to import the bookmark file into your Tor browser.
+
+We can also export Bookmarks from am old version of **Mobile Tor(on Android)** and Import in a New version of the mobile Tor (on Android) following the steps below.
```

Comment:
typo: from _an_ old
and replace "Mobile Tor" to Tor Browser for Android (it's the official name).